src/Services/slidesServices.js

```diff
@@ -1,42 +1,51 @@
 import axios from "axios";
```

Comment:
all this code should be on slideService instead of slidesServices (because its only one service)

Reply:
please delete this file

Reply:
Fixed

Reply:
not fixed yet 
![image](https://user-images.githubusercontent.com/33396785/142467597-11b3cd5e-a2d1-4f87-99e0-d4f73e6a333c.png)

Reply:
Fixed, sorry i miss that